tests/tests.js

```diff
@@ -75,4 +75,48 @@ exports.defineManualTests = function (contentEl, createActionButton) {
       alert("Authentication invalid " + JSON.stringify(err));
     }
   });
+
+  createActionButton("Save secret", function () {
```

Comment:
We could do these as automatic tests too right? Would be good to test that feature in CI.

Reply:
We cant't.